app/services/course/assessment/question/programming/cpp/cpp_package_service.rb

```diff
@@ -0,0 +1,248 @@
+# frozen_string_literal: true
+class Course::Assessment::Question::Programming::Cpp::CppPackageService < \
```

Comment:
@zhengwei143 we prefer to split the PR into smaller pieces, so that they can be reviewed and merged piece by piece, it will also be easier for you to only update a small piece.

I think you don't need to include the full multiple file implementation in this PR, but the design of the multiple files might affect the service you pushed here.